editor-extension-collection/shopify.extension.toml.liquid

```diff
@@ -0,0 +1,13 @@
+[[extensions]]
+name = "t:collection_one_name"
+type = "editor_extension_collection"
```

Comment:
I think we should exclude `editor` from the type name. Possibly even `module` to `extension`, if we really trying to generically template this. 

the use case of course is specifically extensions in the editor for now, but as a module type this is more versatile going forward.

Reply:
Hey John 👋🏽 thanks for the feedback. So the typename here is the name we all aligned on in the tech doc. It was reviewed by @nickwesselman @deannatron and shared with the build ux team. You can view the reasoning behind it in that doc. That being said, we agreed that editor naming was sufficient

Reply:
The recommendation was that the name be inclusive of themes and POS as well, both of which have editors too, in case they want to leverage the same mechanism for surfacing collections to merchants in the editor, with the same pattern, so the recommendation was not to be specific to the checkout and accounts editor.

Reply:
Makes sense. Casts a decent sized net. I still don't think we need to even specify anything to with editors, but there's at least good reasons.

thanks for the context both!